src/Memo/MemoFactory.php

```diff
@@ -25,7 +26,8 @@ public static function create(Table $table, EncoderInterface $encoder): ?MemoInt
         }
         $memoFilepath = $fileInfo['dirname'].DIRECTORY_SEPARATOR.$fileInfo['filename'].$memoExt;
         if (!file_exists($memoFilepath)) {
-            return null; //todo create file?
+            $memo_creator = MemoCreatorFactory::create($table);
```

Comment:
if the DB does not have memo fields, then a memo file will still be created. It is not right. TableWriter should be responsible for creating files.  MemoFactory is part of TableReader

Reply:
Hmm, you're right. Without that line changed in `MemoFactory` it _is_ creating the Memo file. I'm not sure why I thought changing that was necessary 🤔 

Anyway, currently `TableWriter` appears to always create an empty Memo file if the `TableType` supports Memos.

`TableEditor` has a nice method for grabbing any set Memo Columns, but `TableWriter` is its own Class that doesn't have access to `TableEditor`'s methods so we cannot currently use that. Would it be appropriate to move that method to a Trait so that both `TableEditor` and `TableWriter` can use it? 

If `TableWriter` had that method, we could have it check whether or not the Memo file needs to be created instead of always creating it.